sdk/search/test-resources.bicep

```diff
@@ -1,22 +1,24 @@
+param isPublicCloud bool = environment().name == 'AzureCloud'
 param searchEndpointSuffix string = 'search.windows.net'
 param storageEndpointSuffix string = 'core.windows.net'
-param location string = '${resourceGroup().location}'
+param location string = isPublicCloud ? 'eastus2' : resourceGroup().location
```

Comment:
I should have caught this in an earlier iteration. Can we set `Location` in the public cloud entry for the CloudConfig in `tests.yml` instead of having it be hardcoded here?